source/install/cloud-changelog.md

```diff
@@ -4,12 +4,73 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-12-15](#release-2022-12-15)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
-- [Release 2022-10-20](#release-2022-10-20)
+
+## Release 2022-12-15
+
+### Highlights
+
+#### Message Priority and Acknowledgments
+ - Added [message priority labels](https://docs.mattermost.com/channels/message-priority.html) to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts (Professional).
+
+#### Global Drafts
+ - Added a centralized Drafts view for draft messages.
+
+#### Annual Cloud Subscriptions
+ - On the purchase modal, admins are now able to buy an annual cloud subscription for a user count that exceeds the current user count.
+ - The **System Console > Billing & Account > Subscriptions** page now reflects if the plan is monthly or annual.
+ - Added the ability to migrate from a monthly to an annual Cloud Professional plan.
```

Comment:
@amyblais  annual cloud subscription has been put on hold due to other new packaging changes that affect Cloud Professional. We don't have a specific timeline yet on when it will be shipped - cc/ @johndavidlugtu